src/main/java/org/prebid/server/proto/openrtb/ext/request/alkimi/ExtImpAlkimi.java

```diff
@@ -0,0 +1,25 @@
+package org.prebid.server.proto.openrtb.ext.request.alkimi;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+@Value
+@Builder(toBuilder = true)
+public class ExtImpAlkimi {
+
+    String token;
+
+    @JsonProperty("bidFloor")
+    BigDecimal bidFloor;
+
+    Integer pos;
+
+    int width;
+    int height;
```

Comment:
Please, lets make it Integer to avoid NPE on unboxing values from calls like `banner.getW() `, also please add empty line between two declarations